connection.go

```diff
@@ -124,6 +126,10 @@ func (c *Connection) ErrorChan() chan error {
 	return c.errorChan
 }
 
+func (c *Connection) ShutdownFinishedCh() <-chan struct{} {
```

Comment:
This should be `ShutdownFinishedChan()` for consistency with `ErrorChan()` right above it

Reply:
https://github.com/blinklabs-io/gouroboros/pull/493/commits/26d75c059ec39d53ec351a8b767c2b4d66a7b4e6